### PR TITLE
DALI Backend tests for CI: identity and rn50_ensemble

### PR DESCRIPTION
**Pull request by szalpal**
Test scripts, that add 2 tests for dali_backend:
1. `identity`
2. `rn50_ensemble`

Please refer to https://github.com/triton-inference-server/dali_backend/pull/3 for more details

I'm not sure, that my convention (running `.sh` scripts in a `test.sh` script) is going to work. Please advise

Signed-off-by: szalpal <mszolucha@nvidia.com>

### CODE REVIEW

File: qa/L0_backend_dali/test.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+DALI_BACKEND_DIR=/data/dali_backend_dir
```
Comment (deadeyegoodwin) [inline qa/L0_backend_dali/test.sh:28]: /data is mounted into the container and contains test models, etc. So don't use it. I would just clone directly into the test directory. So just use  `pwd`/dali_backend

File: qa/L0_backend_dali/test.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
```
Comment (deadeyegoodwin) [inline qa/L0_backend_dali/test.sh:29]: Add a check to make sure DALI_BACKEND_REPO_TAG is defined. If not just echo an appropriate message and then "exit 1".

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: qa/L0_backend_dali/test.sh (hunk 1 above)
Comment (szalpal) [reply qa/L0_backend_dali/test.sh]: Done

File: qa/L0_backend_dali/test.sh (hunk 2 above)
Comment (szalpal) [reply qa/L0_backend_dali/test.sh:34]: Done

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**